string username
string password
---
bool auth 
